src/app/loadout/selectors.ts

```diff
@@ -122,3 +128,16 @@ export const previousLoadoutSelector =
     }
     return undefined;
   };
+
+export const selectedLoadoutStore = createSelector(
+  sortedStoresSelector,
+  currentStoreSelector,
+  (rootState: RootState) => rootState.loadouts.selectedLoadoutStoreId,
+  (stores, currentStore, selectedLoadoutStoreId): DimStore => {
+    const defaultStore = currentStore || stores[0];
```

Comment:
I always want to return a store of some type. So I went for current first, followed by the user preference. Happy to change this as folks think it makes sense.